README.md

```diff
@@ -606,6 +606,9 @@ curl -X 'POST' 'http://127.0.0.1:8080/v1/query' -H 'accept: application/json' -H
 
 Web page with Swagger UI has the standard `/docs` endpoint. If the service is running on localhost on port 8080, Swagger UI can be accessed on address `http://localhost:8080/docs`.
 
+It is also possible to access Redoc page with three-panel, responsive layout. This page uses `/redoc` endpoint. Again, if the service is running on localhost on port 8080, Redoc UI can be accessed on address `http://localhost:8080/redoc`.
```

Comment:
can be accessed **the** on address**:**

Reply:
well. Nevermind, in the other examples like the one right above, it's written the same way